### DISCUSSION TITLE
Concatenate/combine payloads in flows ?

### DISCUSSION THREAD

**Discussion by battosai30** [directus/directus#16075] [Archive]
Hi ! 

Simple question I can't find an answer : in a flow I "just" want to combine 2 informations in 1.

Example : 

Trigger on item created in a collection -> call a webhook which gives information A and information B -> do A + B and insert it  in a new item in another collection.

I succesfully map single information to single field, but I can't concatenate them.

Thanks !

**Comment by rijkvanzanten** [comment]
@battosai30 I'd recommend using the "Script" operation. That allows you to write a small bit of javascript to modify that data however you want 🙂

**Comment by battosai30** [comment]
I found it just after asking here and after 1 hour searching ... Pfff xD

We agree that there is no ducumentation yet about it ?

**Reply by rijkvanzanten** [reply]
Soon! https://github.com/directus/docs/pull/163

**Reply by battosai30** [reply]
Cool :)
